Solve Issue for Subscribe Box and Add Some CSS for Layout

mr-vijaychauhan opened the pull request:
# Description

First I remove from repeated `./horizontal-slim-10_7.css` from `<body> `and include inside `<head>`. First I solved Issue for subscribed box overlap your character and not looking good and added css for layout and also added responsive css for mobile friendly. I added below screenshot so you understand very well.

*Note : Automatically format of index.html files because I am using VS CODE.

Fixes # (issue)

## Solve Issue for Subscribe Box to overlap to your character image.

# How Has This Been Tested? 

I Upload below Screenshot of Before and After of index.html layout to Easily understanad what I did.

- [ ] Before
![Before__Layout](https://user-images.githubusercontent.com/63604585/96900121-5e5e4200-14af-11eb-845c-2ffae63e3c1e.png)

- [ ] After
![After__Layout](https://user-images.githubusercontent.com/63604585/96900161-69b16d80-14af-11eb-838d-becc38d46566.png)
![PC_for Subscribe Modal](https://user-images.githubusercontent.com/63604585/96900168-6ae29a80-14af-11eb-8c22-6e7e4027c90a.png)
![Responsive_Layout](https://user-images.githubusercontent.com/63604585/96900170-6b7b3100-14af-11eb-9d7c-2c2dbbe3be21.png)
![Responsive__Layout](https://user-images.githubusercontent.com/63604585/96900171-6c13c780-14af-11eb-911a-d23e6108ab33.png)

eddiejaoude commented:
Thank you @mr-vijaychauhan 👍 , looks like great improvements 🤓 

Please can you do the formatting of the code in its own separate PR otherwise it is impossible to see the actual code changes.

mr-vijaychauhan commented:
> Thank you @mr-vijaychauhan 👍 , looks like great improvements 🤓
> 
> Please can you do the formatting of the code in its own separate PR otherwise it is impossible to see the actual code changes.

Thank You Sir to Replied. I did Changes as You Says. Please Check this Changes.

mr-vijaychauhan commented:
Final Changes Done

eddiejaoude commented on index.html:
```diff
@@ -23,6 +23,57 @@
+    }
+
+    /* Responsive layout */
+    @media (max-width: 800px) {
```
I think it might be best to use the bootstrap responsive classes?

eddiejaoude commented:
Thank you 👍 . 

The last word on a new line by itself looks a bit odd, can it be made wider or smaller?

<img width="1207" alt="Screenshot 2020-10-25 at 22 14 59" src="https://user-images.githubusercontent.com/624760/97120568-e9d60e00-170f-11eb-86b1-08dc4829cd6c.png">

The form breaks outside the card

<img width="749" alt="Screenshot 2020-10-25 at 22 15 10" src="https://user-images.githubusercontent.com/624760/97120577-fa868400-170f-11eb-8012-014deeb2ba5d.png">

mr-vijaychauhan commented:
Hi Sir, 

# Description

I added bootstrap class for Subscribe box as well as Responsive Devices. I Fixed Problem Form breaks outside the card. I added Bootstrap Class For Your Content Justify. I tried to fix your content overlapping your character images.

- [ ] Before (Without Justify Content)
![Screenshot (52)](https://user-images.githubusercontent.com/63604585/97157659-5ef02480-179e-11eb-88fc-a1529c76ddab.png)

- [ ] After (With Justify Content and Fix Problem to Overlapping Character Image)
![Screenshot (53)](https://user-images.githubusercontent.com/63604585/97157752-821ad400-179e-11eb-82ea-a27a5661125e.png)

- [ ] I cann't fix this Overlapping Problem in Mobile Device because Your Character Inside Your Body
![Screenshot (54)](https://user-images.githubusercontent.com/63604585/97157848-ab3b6480-179e-11eb-96f8-898c55e22920.png)

I tried My best to Solve Your Problem.

Thanks  !

eddiejaoude commented:
This looks much better now 👍 . Thank you 🤓

eddiejaoude reviewed (approved):
Looks great 👍
